vacasa/connect/connect.py

```diff
@@ -1,12 +1,11 @@
 """Vacasa Connect Python SDK."""
+from retry import retry
 import hashlib
 import hmac
-from typing import Optional
-from urllib.parse import urlparse, urlunparse
-
-import backoff
 import pendulum
 import requests
+from typing import Optional
+from urllib.parse import urlparse, urlunparse
 
```

Comment:
You're messing up the feng shui of the imports. The PEP8 guidance on imports says:

> Imports should be grouped in the following order:
> 1. Standard library imports.
> 2. Related third party imports.
> 3. Local application/library specific imports.
>
> You should put a blank line between each group of imports.

https://www.python.org/dev/peps/pep-0008/#imports

Reply:
I did a reformat code, and it didn't seem to find any issues, but I can fix this

Reply:
👍 https://www.jetbrains.com/help/pycharm-edu/optimizing-imports.html

Reply:
This is what optimize imports gives me:
import hashlib
import hmac
from typing import Optional
from urllib.parse import urlparse, urlunparse

import pendulum
import requests
from retry import retry